src/Unichain.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity >=0.8.0;
+
+library Unichain {
+
+    /******************************************************************************************************************/
+    /*** Token Addresses                                                                                            ***/
+    /******************************************************************************************************************/
+
+    address internal constant SUSDS = 0xA06b10Db9F390990364A3984C04FaDf1c13691b5;
+    address internal constant USDC  = 0x078D782b760474a361dDA0AF3839290b0EF57AD6;
+    address internal constant USDS  = 0x7E10036Acc4B56d4dFCa3b77810356CE52313F9C;
+
+    /******************************************************************************************************************/
+    /*** Bridging Addresses                                                                                         ***/
+    /******************************************************************************************************************/
+
+    address internal constant CCTP_TOKEN_MESSENGER = 0x4e744b28E787c3aD0e810eD65A24461D4ac5a762;
+    address internal constant SKY_GOV_RELAY        = 0x3510a7F16F549EcD0Ef018DE0B3c2ad7c742990f;
+    address internal constant TOKEN_BRIDGE         = 0xa13152006D0216Fe4627a0D3B006087A6a55D752;
+
+    /******************************************************************************************************************/
+    /*** PSM Addresses                                                                                              ***/
+    /******************************************************************************************************************/
+
+    address internal constant PSM3 = 0x7b42Ed932f26509465F7cE3FAF76FfCe1275312f;
+
+    /******************************************************************************************************************/
+    /*** Spark Liquidity Layer Addresses                                                                            ***/
+    /******************************************************************************************************************/
+
+    address internal constant ALM_CONTROLLER  = 0x8Ca42889FF592fE694dBD319a8e002a6054bFF5C;
+    address internal constant ALM_PROXY       = 0x90E795522421Fb24b4053e68A1Aed2e1090c7839;
+    address internal constant ALM_RATE_LIMITS = 0x7F92C47326084E96A1C0D9BCc8F4F3B76A67a0BA;
+    address internal constant ALM_FREEZER  = 0x90D8c80C028B4C09C0d8dcAab9bbB057F0513431;
+    address internal constant ALM_RELAYER  = 0x8a25A24EDE9482C4Fc0738F99611BE58F1c839AB;
+    address internal constant ALM_RELAYER2 = 0x8Cc0Cb0cfB6B7e548cfd395B833c05C346534795;
```

Comment:
Need to put these in their own blocks

Reply:
Verified that these addresses match the other domains

Reply:
seperated